tests/cli/BuildCest.php

```diff
@@ -48,9 +48,28 @@ public function usesTypehintsWherePossible(CliGuy $I, Scenario $scenario)
         $I->seeInThisFile('class CliGuy extends \Codeception\Actor');
         $I->seeInThisFile('use _generated\CliGuyActions');
         $I->seeFileFound('CliGuyActions.php', 'tests/support/_generated');
-        $I->openFile(codecept_root_dir('tests/support/CliHelper.php'));
-        $I->seeInThisFile('public function grabFromOutput($regex): int');
-        $I->seeInThisFile('return $match[1]');
+        $I->seeInThisFile('public function grabFromOutput(string $regex): int');
+    }
+
+    public function generatedUnionReturnTypeOnPhp8(CliGuy $I, Scenario $scenario)
+    {
+        if (PHP_MAJOR_VERSION < 8) {
```

Comment:
I'm wondering if Codeception supports [`@requires`](https://phpunit.readthedocs.io/en/9.5/incomplete-and-skipped-tests.html#skipping-tests-using-requires) annotation. If so, I think it would be good to use it, since it is cleaner than this conditional and in the future it can be replaced with its homonymous attribute in php 8

Reply:
I tried it, it has no effect.

Reply:
Oh 😞 it would be great to have that. Thanks.